bin/sidekiq

```diff
@@ -36,6 +36,9 @@ rescue => e
     STDERR.puts e.backtrace.join("\n")
   else
     cli.handle_exception e
+
+    warn e.message
+    warn e.backtrace.join("\n")
```

Comment:
This is borrowed from the most recent `main` version of `bin/sidekiq` [here](https://github.com/sidekiq/sidekiq/blame/main/bin/sidekiq#L34-L35).